include/insns/Zicbom.h

```diff
@@ -0,0 +1,70 @@
+//
+// _Zicbom_h_
+//
+// Copyright (C) 2017-2023 Tactical Computing Laboratories, LLC
+// All Rights Reserved
+// contact@tactcomplabs.com
+//
+// See LICENSE in the top level directory for licensing details
+//
+
+#ifndef _SST_REVCPU_ZICBOM_H_
+#define _SST_REVCPU_ZICBOM_H_
+
+#include "../RevInstHelpers.h"
```

Comment:
Do you need the relative header locations?

Reply:
matches the other insn implementation headers